### PR TITLE
New K website

### PR DESCRIPTION
**Pull request by shd101wyy**
Screenshot from 2020-08-13 22-25-01

Demo: https://shd101wyy.github.io/k

Then Jenkins task is trying to build the static website in `./web/public_content` directory, then move everything inside `./web/public_content` into `./`.

### CODE REVIEW

File: web/pages/tutorials.md
Comment (ehildenb) [inline web/pages/tutorials.md]: Here

**Comment by dwightguth** [comment]
The static content is in serious need of making sure the out of date stuff is removed; that was one of the primary reasons we wanted to update the website.

**Comment by ehildenb** [comment]
@shd101wyy we have decided a few things at the meeting, the gist of it is "Only keep up-to-date information about the current state of K." What that means is:

-   Please remove all links to the `tool/run` try it online tool for now.
-   Put the links to the tutorial videos at the _bottom_ of the page, and put some text telling people they are out of date.
-   Please remove all pages with publications/people/workshops (eg: http://www.kframework.org/K11/).

That should be a good start for now.

**Comment by shd101wyy** [comment]
Hi @ehildenb I have updated the website. Please see the demo here: https://shd101wyy.github.io/k/index.html 
Please let me know if I need to make any further changes. Thank you!

**Comment by ehildenb** [comment]
@shd101wyy for the moment, let's also remove the "Projects", "News", "Events", and "Funding" pages from the site, since they are all out of date.

Also, please provide a link (in the left pane) called "User Documentation", which links to the file `pending-documentation.md` (in the K repo root), and another link called "Builtins" which links to `k-distribution/include/builtin/index.html` (generated from `k-distribution/include/builtin/README.md`).

File: web/pages/downloads.md
Comment (ehildenb) [inline web/pages/downloads.md]:
```suggestion
```

We don't want to distribute old versions of K anymore.

**Comment by shd101wyy** [comment]
Hi @ehildenb , I have removed the out of date pages and added the User Documentation and Builtins pages. I also supported the syntax highlighting for K. Please take a look at the demo website when you have time. Thanks a lot!

https://shd101wyy.github.io/k

A page that contains example of syntax highlighting for K: https://shd101wyy.github.io/k/k-distribution/include/kframework/builtin/domains/

**Review by ehildenb — APPROVED** [review]

**Comment by ehildenb** [comment]
Wow, this looks really great! Thanks @shd101wyy . I'm approving, but I believe you need @dwightguth 's approval to merge it because he's code-owner on some files you touched.

My thoughts are that any more fixes can be made note of and cleaned up in a follow-up PR within the next few weeks.

**Review by dwightguth — APPROVED** [review]

File: k-distribution/include/kframework/builtin/README.md
Comment (nishantjr) [inline k-distribution/include/kframework/builtin/README.md:23]: This should be `.md`? Also any reason this is an absolute path?

Comment (ehildenb) [reply k-distribution/include/kframework/builtin/README.md:23]: @nishantjr please open a new issue or a PR instead.

**Pull request merged**
